snap/snapcraft.yaml

```diff
@@ -29,8 +29,39 @@ apps:
       - network-bind
 
 parts:
+
+  lensfun:
+    plugin: cmake
+    source: https://github.com/lensfun/lensfun.git
+    source-commit: v0.3.2
+    configflags:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_INSTALL_PREFIX=/usr
+      - -DBINARY_PACKAGE_BUILD=On
+
+    build-packages:
+      - python3
+      - libglib2.0-dev
+      - libpng-dev
+      - doxygen
+      - python3-docutils
+
+    override-build: |
+      snapcraftctl build
+      # run lensfun-update-data and update lens data
+      PYTHONPATH=/usr/lib/python3.6/site-packages/ lensfun-update-data
+      mv -v /var/lib/lensfun-updates/version_1/*.xml ${SNAPCRAFT_PART_INSTALL}/usr/share/lensfun/version_1
+      rm -rf /var/lib/lensfun-updates/version_1
+
+    override-stage: |
+      snapcraftctl stage
+      # not sure why this is needed? is ldconfig step missing?
+      cd "${SNAPCRAFT_STAGE}"/usr/lib && ln -sf x86_64-linux-gnu/liblensfun.so .
```

Comment:
Indeed, this shouldn't be needed, the triplet dir should be added to `LD_LIBRARY_PATH`. Is it because you're using `-DLensFun_LIBRARY=${SNAPCRAFT_STAGE}/usr/lib/liblensfun.so` below? If you used the triplet dir there, would this be unnecessary?

Reply:
I will check this.  I went around the horn trying to get darktable to find liblensfun.  I thought having staged liblensfun would have been enough, but cmake wasn't looking in that directory - it only looks at "/usr/lib" (ish), but possibly only in its local version

Now upon reflection this isn't arch independent as written - nor if ported to the CMAKE line below - I'll see what I can do.

Reply:
rewritten with SNAPCRAFT_ARCH_TRIPLET